Makefile

```diff
@@ -84,6 +90,14 @@ build-darwin:
 container: build
 	docker buildx build --platform="linux/$(ARCH)" --no-cache -t $(IMAGE_TAG) -f Dockerfile --progress=plain .
 
+.PHONY: arc-conformance-container
+arc-conformance-container: docker-buildx-builder
+	docker buildx build \
+	--no-cache \
+	--platform="linux/$(ARCH)" \
```

Comment:
you'll need something like this here https://github.com/Azure/azure-workload-identity/blob/main/Makefile#L124 so you can do multiple arch

Reply:
I see that you are doing this ADO. Is nothing here going to call this?

Reply:
We do have [container-all](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/c5723698ab7fd4ce0ef36baf16e29bec56d518a6/Makefile#L112) target which we use for e2e and local development. For conformance test imo we can just use default ARC from makefile i.e. amd64. We we do want to support multiple arc, ergo in ADO we do this.

Reply:
If we're building multi arch images, then we also need a target to create `docker manifest` and push it?

Reply:
We can. But do we need it for local testing? We can test it with default arch for local? in ADO it's already taken care of.

Reply:
I think the targets should have all options. Is there a reason for doing this in ADO pipeline instead of having make targets and reusing it in ADO?

Reply:
I followed our [provider pipeline](https://dev.azure.com/AzureContainerUpstream/Kubernetes/_git/ado-build-definitions?path=/pipelines/secrets-store-csi-driver-provider-azure.build.yml&version=GBmaster&line=37&lineEnd=38&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents).

Reply:
Got it! I think it'll still be good to have a target here in case anyone wants to build it, similar to the `container-all` we have for the provider image.

Reply:
added